back/app/controllers/web_api/v1/activities_controller.rb

```diff
@@ -1,37 +1,31 @@
 # frozen_string_literal: true
 
 class WebApi::V1::ActivitiesController < ApplicationController
-  before_action :set_post_type_and_id
-  skip_after_action :verify_policy_scoped
   skip_before_action :authenticate_user, only: %i[index]
 
   def index
-    activities = ActivitiesFinder.new(
-      finder_params,
-      includes: :user,
-      scope: policy_scope(Activity)
-    ).find_records
-    paginated_activities = paginate activities
+    @activities = policy_scope Activity
+    @activities = @activities.where(user_id: params[:user_ids]) if params[:user_ids]
+    @activities = @activities.where(project_id: params[:project_ids]) if params[:project_ids]
+
+    @activities = case params[:sort]
+    when 'acted_at'
+      @activities.order(acted_at: :asc)
+    when '-acted_at'
+      @activities.order(acted_at: :desc)
+    when nil
+      @activities
+    else
+      raise 'Unsupported sort method'
+    end
+
+    paginated_activities = paginate @activities
```

Comment:
Should add includes after pagination to avoid performance issues:

```suggestion
    paginated_activities = paginate @activities
    paginated_activities = paginated_activities.includes(:user)
```

Reply:
@jamesspeake You mean in preference to where it is now?
```
render json: linked_json(
      paginated_activities,
      WebApi::V1::ActivitySerializer,
      params: jsonapi_serializer_params,
      include: %i[user item]
    )
    ```